code/qcommon/cm_load.c

```diff
@@ -748,6 +748,9 @@ int CM_LoadSubBSP(const char *name)
 
     CM_FloodAreaConnections(cm);
 
+    // write the BSP name into the clipmap struct so that it could be reused.
+    Q_strncpyz(cm->name, name, sizeof(cm->name));
```

Comment:
Just wondering, because the name was never initially set: is this actually properly cleared at one point?

Reply:
Yes, on SV_SpawnServer (every "slow" map load), the clipmap array is cleared out.

https://github.com/sof2plus/sof2plus-engine/blob/b9b2b0fce9e82d6fbc9e42cbf20595042b93407f/code/server/sv_init.c#L449